src/functions/accent.js

```diff
@@ -0,0 +1,246 @@
+            positionType: "firstBaseline",
+            children: [
+                {type: "elem", elem: body},
+                {type: "kern", size: -clearance},
+                {type: "elem", elem: accentBody},
+            ],
+        }, options);
+
+    } else {
+        accentBody = stretchy.svgSpan(group, options);
+
+        accentBody = buildCommon.makeVList({
+            positionType: "firstBaseline",
+            children: [
+                {type: "elem", elem: body},
+                {type: "elem", elem: accentBody},
+            ],
+        }, options);
+
+        // TODO: pass "svg-align" to stretchy.svgSpan
+        const styleSpan: domTree.span =
+            (accentBody: any).children[0].children[0].children[1];
+        styleSpan.classes.push("svg-align");  // text-align: left;
+        if (skew > 0) {
+            // Shorten the accent and nudge it to the right.
+            styleSpan.style.width = `calc(100% - ${2 * skew}em)`;
+            styleSpan.style.marginLeft = (2 * skew) + "em";
+        }
+    }
+
+    const accentWrap =
+        buildCommon.makeSpan(["mord", "accent"], [accentBody], options);
+
+    if (supsubGroup) {
+        // Here, we replace the "base" child of the supsub with our newly
+        // generated accent.
+        supsubGroup.children[0] = accentWrap;
+
+        // Since we don't rerun the height calculation after replacing the
+        // accent, we manually recalculate height.
+        supsubGroup.height = Math.max(accentWrap.height, supsubGroup.height);
+
+        // Accents should always be ords, even when their innards are not.
+        supsubGroup.classes[0] = "mord";
+
+        return supsubGroup;
+    } else {
+        return accentWrap;
+    }
+};
+
+const mathmlBuilder = (group, options) => {
+    let accentNode;
+    if (group.value.isStretchy) {
+        accentNode = stretchy.mathMLnode(group.value.label);
+    } else {
+        accentNode = new mathMLTree.MathNode(
+            "mo", [mml.makeText(group.value.label, group.mode)]);
+    }
+
+    const node = new mathMLTree.MathNode(
+        "mover",
+        [mml.buildGroup(group.value.base, options), accentNode]);
+
+    node.setAttribute("accent", "true");
+
+    return node;
+};
+
+// Accents
+defineFunction({
+    type: "accent",
+    names: [
+        "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+        "\\check", "\\hat", "\\vec", "\\dot",
+        "\\widehat", "\\widetilde", "\\overrightarrow", "\\overleftarrow",
+        "\\Overrightarrow", "\\overleftrightarrow", "\\overgroup",
+        "\\overlinesegment", "\\overleftharpoon", "\\overrightharpoon",
+    ],
+    props: {
+        numArgs: 1,
+    },
+    handler: (context, args) => {
+        const base = args[0];
+
+        const isStretchy = !utils.contains([
+            "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+            "\\check", "\\hat", "\\vec", "\\dot",
+        ], context.funcName);
```

Comment:
For performance, consider moving the array of stretchy function names outside the function so it doesn't get re-created on each invocation. 

According to https://jsperf.com/static-arrays-inside-functions/1 moving the array out of the function will make this operation twice as fast.

Reply:
A regexp (outside the function) seems like it would also be a good choice here instead of an array of strings.